src/components/accessories/dashboard/dashboardContent/filter/DashboardFilter.tsx

```diff
@@ -129,39 +129,35 @@ export const DashboardFilter: FC<IOwnProps> = ({ onPeriodChange }) => {
             </ToggleButton>
           </ToggleButtonGroup>
         )}
-      </div>
-      <div className="filter__actions">
-        <span>{period}</span>
-        {view === "range" ? (
-          <DateRangeField
-            fieldName="period"
-            isValid={true}
-            fieldValue={dateRange}
-            format="dd/MM/YYY"
-            onChange={handleDateRangeChange}
-            TextFieldComponent={(props) => (
-              <IconButton onClick={onIconClickHandler}>
-                <CalendarTodaySharp />
-              </IconButton>
-            )}
-            open={open}
-          />
-        ) : (
-          <DateField
-            fieldName="period"
-            isValid={true}
-            errorText=""
-            label=""
-            fieldValue={dateRange[0]?.toISOString() ?? ""}
-            format="dd/MM/YYY"
-            onChange={handleDateChange}
-            TextFieldComponent={(props) => (
-              <IconButton onClick={onIconClickHandler}>
-                <CalendarTodaySharp />
-              </IconButton>
-            )}
-            open={open}
-          />
```

Comment:
Hi @axelpezzo ! The DesktopRangePicker doesn't allow to pick the same date(I think) ! By removing the DateField, we won't be anymore able to select a single date(did you try it ?)